services/src/main/java/org/keycloak/services/cors/DefaultCors.java

```diff
@@ -43,16 +43,19 @@ public class DefaultCors implements Cors {
     private final HttpRequest request;
     private final HttpResponse response;
     private ResponseBuilder builder;
+    private final String defaultAllowHeaders;
     private Set<String> allowedOrigins;
     private Set<String> allowedMethods;
+    private Set<String> allowedHeaders;
     private Set<String> exposedHeaders;
 
     private boolean preflight;
     private boolean auth;
 
-    DefaultCors(KeycloakSession session) {
+    DefaultCors(KeycloakSession session, String defaultAllowHeaders) {
```

Comment:
```suggestion
    DefaultCors(KeycloakSession session, Set<String> defaultAllowHeaders) {
```

Is it possible to accept a `Set` of headers? In order to execute `String.format("%s, %s",` only at one place -> `DefaultCors`. The manageability would be better.